README.md

```diff
@@ -91,63 +90,51 @@ The configuration parameters are provided through environment variables:
   export TOKEN_ISSUER=http://localhost:9080/auth/realms/test-smart
   ```
 
-- `ACCESS_CHECKER`: The access-checker to use. Each access-checker has a name
-  (see [plugins](plugins) for details) and this variable should be set to the
-  name of the plugin to use. For example, to use one of the sample plugins
-  include one of:
+- **AccessChecker**: As mentioned above, access-checkers can be provided as
+  plugins and easily swapped. Each access-checker has a name (see
+  [plugins](plugins) for details) and `ACCESS_CHECKER` variable should be set to
+  this name. 
+  
+  For example, the two plugins that are provided in this repository, can be selected by either of:
 
   ```shell
   export ACCESS_CHECKER=list
   export ACCESS_CHECKER=patient
   ```
+  NOTE: The above are just samples provided to demonstrate the concept of a pluggable access-checker. Custom access-checkers can be implemented to support common patterns such as use of `Location`, `Care Team` or `General Practitioner` to return the conceptual list of Patients for an authenticated user
 
-  For more information on how access-checkers work and building your own, see
-  [Understanding access checker plugins](https://github.com/google/fhir-gateway/wiki/Understanding-access-checker-plugins).
-
-- `ALLOWED_QUERIES_FILE`: A list of URL requests that should bypass the access
-  checker and always be allowed.
-  [`AllowedQueriesChecker`](https://github.com/google/fhir-gateway/blob/main/server/src/main/java/com/google/fhir/gateway/AllowedQueriesChecker.java)
+- **AllowedQueriesChecker**: There are URL requests that the server can allow
+  without going through an access checker.
+  [`AllowedQueriesChecker`](https://github.com/google/fhir-access-proxy/blob/main/server/src/main/java/com/google/fhir/proxy/AllowedQueriesChecker.java)
   is a special `AccessChecker` that compares the incoming request with a
-  configured set of allowed-queries. The intended use of this checker is to
-  override all other access-checkers for certain user-defined criteria. The user
-  defines their criteria in a config file and if the URL query matches an entry
-  in the config file, access is granted. An example of this is
-  [`hapi_page_url_allowed_queries.json`](https://github.com/google/fhir-gateway/blob/main/resources/hapi_page_url_allowed_queries.json).
-  To use this file with `ALLOWED_QUERIES_FILE`:
+  configured set of allowed-queries. 
+  
+  The intended use of this checker is to override all other access-checkers for certain user-defined criteria. The user defines their criteria in a config file and if the URL query matches an entry in the config file, access is granted. An example of this is:
```

Comment:
Have you run `mvn package` on your PR? There are still long lines in this file which should have been taken care of with `mvn package`.